src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+// import { event } from 'cypress/types/jquery';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const sortMovies:Movie[] = moviesFromServer.filter(
+    (movie) => movie.title.toLowerCase().includes(query.toLowerCase().trim())
+    || movie.description.toLowerCase().includes(query.toLowerCase().trim()),
+  );
+
+  // console.log(sortMovies);
```

Comment:
Remove commented console.log statement, as it's not necessary for the final code.